src/azure-cli/azure/cli/command_modules/marketplaceordering/generated/_params.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('term show') as c:
+        c.argument('publisher', type=str, help='Publisher identifier string of image being deployed.',
+                   id_part='child_name_1')
+        c.argument('offer', type=str, help='Offer identifier string of image being deployed.', id_part='child_name_2')
+        c.argument('plan', type=str, help='Plan identifier string of image being deployed.', id_part='child_name_3')
```

Comment:
I think `--ids` will fail without the `name` id_part.

Reply:
feedback to codegen also, the team is working on fixing.